markdown/usage/nf_core_tutorial.md

```diff
@@ -365,6 +365,13 @@ That way, you can be confident that everything will work when you fork or open a
 This service is free to use.
 To set it up, visit [https://hub.docker.com](https://hub.docker.com) and link your personal GitHub repository.
 
+To set up an automated docker container build for the master and the dev branches, as well as for all releases, go to the builds tab of your docker hub repository. There, select build configurations. The source repository should be set to your GitHub repository. Three build rules should be set:
+
+* Master branch build rule (comes by default). Set the "Source Type" to "branch", "Source" to "master" and "Docker tag" to "latest".
+* dev branch build rule. Set the "Source Type" to "branch", "Source" to "dev" and "Docker tag" to "dev".
+* Tag build rule: this will build a container for your pipeline releases, and should follow the scenario "Match versions". Set the "Source Type" to "Tag", "Source" to "/^[0-9.]+$/" and "Docker Tag" to "release-{sourceref}".
```

Comment:
```suggestion
* Tag build rule: this will build a container for your pipeline releases, and is similar to the scenario _"Match versions"_. Set the _Source Type_ to `Tag`, _Source_ to `/^[0-9.]+$/` and _Docker Tag_ to `{sourceref}`.
```

NB: I think it's just `{sourceref}`, not `release-{sourceref}` that we want..

Reply:
ok, sorry I thought it was the other one. Thanks for the suggestion!

Reply:
No worries! I think the `release-` is just a static string, so then the dockhub tags become eg. `nfcore/bcellmagic:release-1.1` instead of just `:1.1`